dune/stuff/common/configuration.hh

```diff
@@ -41,20 +45,25 @@ class Request {
   std::string validator;
 
 public:
-  Request(const int _line, const std::string _file, const std::string _key,
-          const std::string _def, const std::string _validator);
+  Request(const int _line,
+          const std::string _file,
+          const std::string _key,
+          const std::string _def,
+          const std::string _validator);
 
   /**
    * \brief less-than operator for Requests
    * \details Compare this' member variables key, def, file, line and validator (in that order) with other's. Returns
    *          true if the first comparison returns true (i.e. this.key < other.key) and returns false if this.key >
-   *          other.key. If neither true nor false is returned in the first comparison (i.e. this.key "==" other.key), the
-   *          next comparison is evaluated similarly. If none of the first four comparisons returns a value, the value of
+   *          other.key. If neither true nor false is returned in the first comparison (i.e. this.key "==" other.key),
+   * the
+   *          next comparison is evaluated similarly. If none of the first four comparisons returns a value, the value
+   * of
    *          this.validator < other.validator is returned.
```

Comment:
These kind of things are stupid but that could be fixed once with some manual work.